osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@nHirsenkorn The vehicle reference point is never rear axis?

This is a definition found together with OEMs over years and also the result of many PEGASUS workshops:

> Ego-Fahrzeug:
> - Position: x[m], y[m], z[m] (Weltkoordinaten; **Bezugspunkt Fahrzeug: Mitte Hinterachse**));

This is a blocking point for VIRES for supporting OSI or not and I'll end discussion here, because nobody of the other tool suppliers, modelers and not all OEMs are participating the discussion.

@raueATconti We need to clarify this before we'll start implementation on our side.